app/lib/address_handling.rb

```diff
@@ -0,0 +1,33 @@
+module AddressHandling
```

Comment:
looks like this is a controller concern, so just thinking whether this would be better (more idiomatically) located in `app/controllers/concerns`?